### DISCUSSION TITLE
failed to transform "/scss/app.scss"

### DISCUSSION THREAD

**Discussion by papanito** [thuliteio/doks#1286] [Q&A] [answered]
While running ` npm run dev`  I get

```
Error: error building site: TOCSS: failed to transform "/scss/app.scss" (text/x-scss): "/home/papanito/website/node_modules/@thulite/core/assets/scss/app.scss:14:1": File to import not found or unreadable: katex/dist/katex.
```

**Comment by h-enk** [comment]
Your repo?

https://github.com/thuliteio/core/blob/main/assets/scss/app.scss

**Reply by h-enk** [reply]
Closed (I see now)

**Reply by papanito** [reply, marked as answer]
Yeah my bad, I try to move my stuff to the new version. I did some custom modifications in the past, so I need to carefully migrate ;-)
